.github/workflows/release-dev.yaml

```diff
@@ -30,15 +30,15 @@ jobs:
 
       - name: Build and push local-artifact-mirror image
         run: |
-          export SHORT_SHA=dev-$(git rev-parse --short=7 HEAD)
+          export SHORT_SHA=dev-${GITHUB_SHA::7}
```

Comment:
this should match what's in the pull-request.yaml version of things

Reply:
can't use the same thing here though as `github.event.pull_request.head.sha` doesn't apply here, no?

Reply:
i guess we could use `GITHUB_SHA` in `pull-request.yaml`?